website/frontend/templates/docs/tags.html

```diff
@@ -391,6 +391,18 @@ <h1 id="basic_variables">Basic Variables</h1>
           <td>_multiartist</td>
           <td>0 if tracks on the album all have the same primary artist, 1 otherwise. (Since Picard 1.3)</td>
         </tr>
+        <tr>
+          <td>_artists_sort</td>
+          <td>The <a href="https://musicbrainz.org/doc/Style/Artist/Sort_Name" title="Style/Artist/Sort Name">Artist Sort Name</a> (Since Picard 1.3)</td>
```

Comment:
Can you add a comment on differentiating this from artistsort? Same for the others.